src/app/modules/main/wallet_section/all_tokens/module.nim

```diff
@@ -95,9 +95,9 @@ method getHistoricalDataForToken*(self: Module, symbol: string, currency: string
 method tokenHistoricalDataResolved*(self: Module, tokenDetails: string) =
   self.view.setTokenHistoricalDataReady(tokenDetails)
 
-method fetchHistoricalBalanceForTokenAsJson*(self: Module, address: string, allAddresses: bool, tokenSymbol: string, currencySymbol: string, timeIntervalEnum: int) =
-  let addresses = if allAddresses: self.addresses else: @[address]
-  self.controller.fetchHistoricalBalanceForTokenAsJson(addresses, allAddresses, tokenSymbol, currencySymbol,timeIntervalEnum)
+method fetchHistoricalBalanceForTokenAsJson*(self: Module, address: string, tokenSymbol: string, currencySymbol: string, timeIntervalEnum: int) =
+  let addresses = if address == "": self.addresses else: @[address]
```

Comment:
It looks to me that `allAddresses` was showing better the intention instead of `address == ""`.